server/valid_text_templates.go

```diff
@@ -0,0 +1,38 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+	"text/template"
+)
+
+// ValidTextTemplateRequest is the request json for validation
+type ValidTextTemplateRequest struct {
+	Template string `json:"template"`
+}
+
+// ValidTextTemplateResponse is the response for template validation
+type ValidTextTemplateResponse struct {
```

Comment:
Maybe an `204 No Content` response (without a body) would make more sense?

Reply:
This is probably best